hack/deploy/configure-values.sh

```diff
@@ -44,7 +44,7 @@ get_vnet_json() {
         local subnet_id
         subnet_id=$(jq -r ".agentPoolProfiles[0].vnetSubnetId" <<< "$aks_json")
         local vnet_id
-        vnet_id=$(echo "$subnet_id" | sed 's|/subnets/[^/]*$||')
+        vnet_id=${subnet_id%/subnets/*}
```

Comment:
This just fixes shellcheck complaining "you can / should do this with variable substitution"